src/main/java/domain/LottoMatch.java

```diff
@@ -0,0 +1,40 @@
+package domain;
+
+import java.util.Arrays;
+
+public enum LottoMatch {
+  DEFUALT_MATCH(0, false, 0),
+  THREE_MATCH(3, false, 5000),
+  FOUR_MATCH(4, false, 50000),
+  FIVE_MATCH(5, false, 1500000),
+  FIVE_BONUS_MATCH(5, true, 30000000),
+  SIX_MATCH(6, false, 2000000000);
+
+  public int winningCounter;
+  public boolean bonusChecker;
+  public int prize;
+
+  private LottoMatch(int winningCounter, boolean bonusChecker, int prize) {
+    this.winningCounter = winningCounter;
+    this.bonusChecker = bonusChecker;
+    this.prize = prize;
+  }
+
+  public static LottoMatch calculateLotto(int winningCounter, boolean bonusChecker) {
+    return Arrays.stream(values()).filter(lottoMatch -> lottoMatch.winningCounter == winningCounter
+        && lottoMatch.bonusChecker == bonusChecker).findFirst().orElse(DEFUALT_MATCH);
+  }
```

Comment:
stream은 연산마다 개행을 주면 가독성을 높일 수 있습니다. :) 

```suggestion
  public static LottoMatch calculateLotto(int winningCounter, boolean bonusChecker) {
    return Arrays.stream(values())
        .filter(lottoMatch -> lottoMatch.winningCounter == winningCounter && lottoMatch.bonusChecker == bonusChecker)
        .findFirst()
        .orElse(DEFUALT_MATCH);
  }
```

Reply:
감사합니다 확실히 개행을 하고나니 가독성이 좋아졌습니다!!  남들이 보기 좋은 코드를 만들도록 노력하겠습니다.